Lab_13_Make_Money/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Make Money Form</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <form id="money-form">
+        <label for="coin-count">How many?</label>
+        <input type="number" id="coin-count" name="coin-count" min="1" required><br><br>
+
+        <label id="coin-type">Which coin?</label>
+        <select id="coin-type" name="coin-type" required>
```

Comment:
Here's the root of the bug. `id` is a unique identifier for exactly one HTML element, but it's being used twice here. The label should use a `for` attribute that will be the same as the input (`select`) id.

```suggestion
        <label for="coin-type">Which coin?</label>
        <select id="coin-type" name="coin-type" required>
```

Reply:
The reason why the js is returning `undefined` is because it's looking at `label.value` and not the selected option's value